src/api/auth/userSignup.ts

```diff
@@ -0,0 +1,12 @@
+export const userSignup = async (formData: FormData) => {
+  const response = await fetch(`/api/auth/user/register`, {
+    method: 'POST',
+    body: formData,
+  });
+
+  if (response.status === 201) {
+    return null;
+  } else {
+    throw new Error('회원가입 실패');
+  }
```

Comment:
statusCode가 `201`일 때 return 하는 `null`이 반환되어 사용되는 값으로 보이지는 않는데요. 사용되지 않는 값을 return 하는 대신 `201`이 아닐 때만 throw를 하게 해주셔도 좋을거 같아요

```suggestion
  if (response.status !== 201) {
    throw new Error('회원가입 실패');
  } 
```